tests/test_cli.py

```diff
@@ -28,6 +28,49 @@ def test_parse_args():
     assert not args.threads
     assert args.variables == ['total_precipitation']
     assert args.land
+    assert not args.area
+
+
+def test_area_values():
+    '''Test if correct area values are expected'''
```

Comment:
```suggestion
    """Test if area argument is parsed correctly."""
```

Quoting from [PEP257](https://www.python.org/dev/peps/pep-0257/): 

> For consistency, always use """triple double quotes""" around docstrings.

and

> The docstring is a phrase ending in a period.